Escape hyphens in manpage

stefanor opened the pull request:
Based on the patch submitted in https://bugs.debian.org/1101406 as the result of running a linter on the manpage.

skirpichev commented on Misc/python.man:
```diff
@@ -183,9 +182,11 @@ useful to inspect global variables or a stack trace when a script
+This also implies \fB\-E\fP, \fB\-P\fP and \fB\-s\fP.
+In isolated mode sys.path contains neither the script's directory nor the
+user's site\-packages directory.
+All PYTHON* environment variables are ignored, too.
```
This looks mostly as reformatting.  Please revert everything, except real change (dash escape in site-packages).

skirpichev commented on Misc/python.man:
```diff
@@ -203,13 +204,13 @@ __debug__; augment the filename for compiled (bytecode) files by
-as the current directory, the script's directory or an empty string. See also the
-\fBPYTHONSAFEPATH\fP environment variable.
+as the current directory, the script's directory or an empty string.
+See also the \fBPYTHONSAFEPATH\fP environment variable.
```
This is just reformatting, please revert.

stefanor replied:
Reverted those bits.

stefanor committed:
Escape hyphens in manpage

Based on the patch submitted in https://bugs.debian.org/1101406 as the
result of running a linter on the manpage.
